build-chat/BuildChat.js

```diff
@@ -176,9 +176,7 @@ async function buildComplete(octokit, owner, repo, runId, options) {
             text: `${name}
 Result: ${build.data.conclusion} | Repository: ${owner}/${repo} | Branch: ${build.data.head_branch} | Authors: ${githubToSlackUsers(githubAccountMap, build.authors, build.degraded).sort().join(', ') ||
                 `None (rebuild)`}
-Build: ${build.buildHtmlUrl}
-Create Issue: ${createIssueLink}
-Changes: ${build.changesHtmlUrl}`,
+<${build.buildHtmlUrl}|Build> | <Create Issue|${createIssueLink}> | <${build.changesHtmlUrl}|Changes>`,
```

Comment:
@chrmarti I disabled the pre-commit hook because it seemed to break the bot but don't forget to compile :)